projects/telemetry/scripts/interface.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
I know you still need to format/lint this but I want to put this comment here as a reminder :p